.lgtm.yml

```diff
@@ -28,8 +28,7 @@ extraction:
             # setup the venv and install packages
             - "python3 -m venv ./fprime-venv"
             - ". ./fprime-venv/bin/activate" 
-            - "pip install ./Fw/Python"
-            - "pip install ./Gds"
+            - "pip install fprime-tools fprime-gds"
```

Comment:
Once these are moved to their own repositories this should be fine, but for now this means that we're not testing changes to fprime-util. Can we install the local versions of fprime-util and the gds until we've broken them out into their own repositories?

Reply:
The above is the LGTM file.  It isn't testing, just building for code analysis.  It doesn't care about using a different version of the tools.  The build will result in the same output, or we broke the build entirely and have to re-release packages anyway.

As a side note, it is likely easier to separate out the packages than it is to fix the internals of a docker VM to work with the correct versioning scheme.  Since we are very close to separating out the packages I think this is more work than we can justify.